handlers/extensions.go

```diff
@@ -50,5 +51,22 @@ func (h *Handler) LoadExtensionFromPackage(w http.ResponseWriter, req *http.Requ
 		extendedEndpoints[output.Router.Path] = output.Router
 	}
 
+	// Run Playground function
+	symRunPlayground, err := plug.Lookup("RunPlayground")
```

Comment:
What's the set of thoughts and intention here? Is there a design spec that speaks to considerations around the playground?

Reply:
I created a discussion so as to not miss the need to address this. Can you address before merge?

Reply:
@kumarabd while you're updating Meshery Arch for database storage details, will you add documentation of considerations around this playground?

Reply:
This was a temporary hack. has been removed now from the master branch and the latest release.